lib/project_types/rails/cli.rb

```diff
@@ -1,11 +1,11 @@
 # frozen_string_literal: true
 module Rails
   class Project < ShopifyCli::ProjectType
+    # hidden_project_type
```

Comment:
Is this defining it as hidden or just a comment? The docs made it seem like this should not be a comment, so I just want to make sure.

Reply:
That is just a comment, I am leaving it as an example for people investigating in the future. But it can be deleted